page_analyzer/templates/url.html

```diff
@@ -42,7 +42,18 @@ <h2 class="mt-5 mb-3">Проверки</h2>
             <th>Дата создания</th>
           </tr>
         </thead>
-        <tbody></tbody>
+        {% for check in checks %}
+        <tbody>
+          <tr>
+            <td>{{ check[0] }}</td>
+            <td>{{ check[2] }}</td>
+            <td>{{ check[3] }}</td>
+            <td>{{ check[4] }}</td>
+            <td>{{ check[5] }}</td>
+            <td>{{ check[6] }}</td>
```

Comment:
wdyt about using `object.property` (like in `url.id` above) instead of index access?